tests/lax_numpy_test.py

```diff
@@ -828,6 +828,21 @@ def testRoundStaticDecimals(self, shape, dtype, decimals, rng_factory):
     self._CompileAndCheck(lnp_fun, args_maker, check_dtypes=check_dtypes,
                           atol=tol, rtol=tol)
 
+  def testOperatorRound(self):
```

Comment:
This test probably needs to be skipped on Python 2, but we'll see what Travis says

Reply:
Travis was happy, it seems.